lib/management/types.ts

```diff
@@ -240,3 +240,104 @@ export enum UserStatus {
   disabled = 'disabled',
   invited = 'invited',
 }
+
+export enum AuthzNodeExpressionType {
```

Comment:
not critical, but note that *TS union string* is consider to be simpler form of enum (unless specific enum feature is required)
e.g for enum string:
![image](https://github.com/descope/node-sdk/assets/10514677/e4b02332-1f41-45a4-851d-2cc6de3a3d0e)


enums are great as well :)

Reply:
Leaving as is because don't want to change `UserStatus` that was there already...